### DISCUSSION TITLE
About Data Reconciliation with Wiki

### DISCUSSION THREAD

**Discussion by candlecao** [DDMAL/linkedmusic-datalake#44] [General]
About Data Reconciliation with Wiki: 
Should we find the class for each object/item/instance in a flattened CSV file as possible? 
For example, I looked into the RDF files from Van: in CantusDB, and see: a chant has its class "<https://www.wikidata.org/wiki/Q23072435>" and in SimssaDB, there is a class "<https://www.wikidata.org/wiki/Q2188189>", etc. @fujinaga @annamorphism
I think we had better have the class mapping in addition to instance mapping.  Thanks for any advice.

**Comment by fujinaga** [comment]
The idea of Data Lakes and Lazy Searching is to do as little as possible when we are collecting data.
It doesn't really matter, at this point, whether we use property, class, or instance URI of Wikidata. 
Our current goal is to put data from multiple databases into our LinkedMusic Datalake in Virtuoso.

Once we have data from several sources, we'll experiment on how we can make queries across data from different sources.

For now, let's concentrate on dumping data into the lake.

**Reply by candlecao** [reply]
OK, Got it. Thank you!
